waku/v2/node/wakunode2.nim

```diff
@@ -789,7 +789,7 @@ proc stopDiscv5*(node: WakuNode): Future[bool] {.async.} =
 
     debug "Successfully stopped discovery v5 service"
 
-proc start*(node: WakuNode) {.async.} =
+proc start*(node: WakuNode, websocketConn: bool = false) {.async.} =
```

Comment:
Is there a reason the websocketConn isn't part of WakuNode object? Seems a bit odd to add another argument to start here but maybe it's fine. It is false by default so that's good regardless for compatibility.

Reply:
fixed

Reply:
I think this still needs to be removed from the `start()` arguments.

Reply:
39ee44c60e9ed111d87a4733e7aadc8efd6119be fixes the same.